packager/packager.js

```diff
@@ -15,24 +15,26 @@ var http = require('http');
 
 var getFlowTypeCheckMiddleware = require('./getFlowTypeCheckMiddleware');
 
-if (!fs.existsSync(path.resolve(__dirname, '..', 'node_modules'))) {
+try {
+  var chalk = require('chalk');
+  var connect = require('connect');
+  var ReactPackager = require('./react-packager');
+  var blacklist = require('./blacklist.js');
+  var launchEditor = require('./launchEditor.js');
+  var parseCommandLine = require('./parseCommandLine.js');
+  var webSocketProxy = require('./webSocketProxy.js');
```

Comment:
Some of the local modules use modules from npm (or they use modules that use modules from npm, etc etc).

There may be a more fine-grained way to do it, but this seemed like the simplest way to maintain the "print this message and bail if a dependency isn't here" idea.